x-pack/plugins/ml/server/models/data_recognizer/modules/security_linux/ml/v3_linux_anomalous_network_activity.json

```diff
@@ -31,32 +21,7 @@
     "time_field": "@timestamp"
   },
   "custom_settings": {
-    "job_tags": {
```

Comment:
@peteharverson Thanks for providing context. My thoughts on the above:
- Afaik we aren't using these fields internally. 
- In terms of searching in the UI, I'm not sure how much of a value searching for v3 jobs is providing. Typically, we see our users upgrading to the latest version of jobs, rather than keeping both old as well as new jobs. Also having versions only for some jobs is a bit odd, given that other jobs (that don't have the version tags) have also undergone updates. I would like to have some versioning for all jobs, but I'd like to discuss this further outside of this PR, given that we are unofficially already several versions along across all our modules.
- I don't understand the value provided by the `euid` field, specifically given that it is retained when the job is cloned. I'd understand if it was a unique identifier for a job so as to distinguish between different copies of the same job, but that not being the case, I don't see a ton of value in it.

Reply:
I'm happy to defer to you on this one @ajosh0504 and remove these `job_tags`. 

The thinking behind `euid` was that this field couldn't (easily) be changed by the user, whereas they are able to completely change the job ID when cloning, so it could be used to track how many of this type of job had been created.

Makes sense to address a way to version all the security jobs outside of this PR.